### ISSUE TITLE
Localization and Phrase filters

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/accessibility-checker#84]
Add support for other languages. Possibly add filters for rule check phrases/keywords.

https://equalizedigital.zendesk.com/agent/tickets/5769

**Comment by oldrup** [comment]
Hi @SteveJonesDev 

I think this is related. I'm experimenting with a Danish translation, and while the descriptions are easily translatable, strings that trigger the warnings like Ambiguous Anchor Text are not. So Accessibility Checker detects strings like "Read more" or "Click here", but not the Danish counter parts "Læs mere" or "Klik her".

Screenshot 1: the content
![image](https://github.com/equalizedigital/accessibility-checker/assets/17141205/af966dc5-5df4-4c67-95d4-88feba1f453f)

Screenshot 2: the warnings
![image](https://github.com/equalizedigital/accessibility-checker/assets/17141205/52de186f-ffa4-4047-b507-b7f4f629a9e4)

Can the strings you check for be moved to the .po file, or is that inefficient, or are they baked into some regexp? 

Warning about "click here" in danish html documents is badly needed, unfortunately 😩

**Comment by SteveJonesDev** [comment]
@pattonwebz,

Since these are now translatable: https://github.com/equalizedigital/accessibility-checker/blob/c8b2c008bc60450f074005941d21eafc3284314b/src/pageScanner/checks/has-ambiguous-text.js#L3-L4

Shouldn't they show up here?
https://translate.wordpress.org/projects/wp-plugins/accessibility-checker/stable/de/default/

For the translations to work, the Regex probably needs changed.
https://github.com/equalizedigital/accessibility-checker/blob/c8b2c008bc60450f074005941d21eafc3284314b/src/pageScanner/checks/has-ambiguous-text.js#L30-L31

```JS
// Use Unicode-aware regex to preserve all letters, not just a-z
text = text.toLowerCase().replace( /[^\p{L}]+/gu, ' ' ).trim();
```

**Comment by pattonwebz** [comment]
We were having issues with translations for a while not being detected on .org. That issue should be resolved. All (or very nearly all) the strings should be translatable. We also have been bundling over 30 different languages in the plugin directly to avoid any lag on translations reaching .org and getting approved by their PTEs.

@oldrup I know it has been a long time since you opened this issue but to get these strings translated you will need to find the wp-content/languages/accessibility-checker/ folder and clear out the .org provided translations file that is on your site which will allow the bundled one in the plugin to take over which should have far more strings translated into your language.

**Comment by oldrup** [comment]
Oh. Thank you, I've completely forgotten all about this. I don't use "click here" in any projects I work on, so. The recommendation will be included in an editor handbook I want to write during summer and fall, so I'll be testing Accessibility Checkers to identify Danish strings then. 

But yeah; translations have come long way the last two years. I assume this is no issue anymore.

Thank you all for your hard work on this important tool 🙏
